### PR TITLE
Added hf

### PR DESCRIPTION
**Pull request by zhaohan-dong**

### CODE REVIEW

**Comment by enochkan** [comment]
@zhaohan-dong please add transformers to `setup.py` as dependency.

File: plexiglass/model.py [hunk 1]
```diff
@@ -1,22 +1,41 @@
...
-               # run_hf_inference()
-               pass
\ No newline at end of file
+    def prompt(self, messages: list) -> ModelResponse:
+        if self.model_type != "hf":
+            response = completion(self.model_name, messages=messages)
+        elif self.model_type == "hf":
+
+            # Text generation
+            input_ids = self._tokenizer.encode(messages[0], return_tensors="pt")
```
Comment (enochkan) [inline plexiglass/model.py]: isn't it easier to use `pipelines` to generate text in the same format for all models? 

https://huggingface.co/docs/transformers/quicktour

**Comment by zhaohan-dong** [comment]
Made changes to use `pipeline` and added `transformers` library. Pending verification on multiple message input.

File: plexiglass/model.py [hunk 2]
```diff
@@ -1,22 +1,26 @@
-from litellm import completion
+from litellm import completion, ModelResponse, Usage, Choices, Message
```
Comment (enochkan) [inline plexiglass/model.py]: `ModelResponse, Usage, .. ` are redundant, no?

**Review by enochkan — APPROVED** [review]

**Comment by enochkan** [comment]
please resolve merge conflict, then I can merge.

**Pull request closed without merging**